src/backend/tests/test-data/reimbursement-requests.test-data.ts

```diff
@@ -232,3 +232,17 @@ export const sharedGiveMeMyMoney: ReimbursementRequest = {
     }
   ]
 };
+
+export const reimbursementMock = {
+  reimbursementId: 'reimbursementMockId',
+  purchaserId: batman.userId,
+  amount: 12,
+  dateCreated: new Date('2023-01-01'),
+  userSubmitted: batman,
+  userSubmittedId: batman.userId
+};
+
+export const updatedReimbursementMock = {
```

Comment:
This is only used in one place, so you don't even need to declare this constant - you can just use the spread operator in the test - that way it is clear what youre doing